components/Modal/index.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AriaModal from 'react-aria-modal';
+import classNames from 'classnames';
+
+import styles from './index.scss';
+
+import Button from '../Button';
+
+export default class Modal extends React.Component {
+
+  static propTypes = {
+    actionText: PropTypes.string,
+    cancelText: PropTypes.string,
+    children: PropTypes.element.isRequired,
+    closeText: PropTypes.string,
+    onAction: PropTypes.func, // This should be a promise, so that onExit can be executed on success (nicer animation)
+    onExit: PropTypes.func.isRequired,
+    open: PropTypes.bool.isRequired,
+    titleText: PropTypes.string.isRequired
+  }
+
+  static defaultProps = {
+    actionText: '',
+    cancelText: '',
+    closeText: 'Close',
+    onAction: () => {},
+    verticallyCenter: true
```

Comment:
This is one of the react-aria-modal props, that we simply spread. We want to have a different default here, than react-aria-modal itself, but if we add it to propTypes, we get a linting error, since we don't use it